recipe/meta.yaml

```diff
@@ -65,18 +146,33 @@ requirements:
     # used in the future along with make when bugs with Pythia8 are fixed
     - {{ compiler('cxx') }}
     - {{ compiler('fortran') }}
-    # - make
+    - make
     - cross-python_{{ target_platform }}  # [build_platform != target_platform]
     - python  # [build_platform != target_platform]
     - sed
     - grep
+    - fastjet-cxx  # [build_platform != target_platform]
+    - emela  # [build_platform != target_platform]
+    - lhapdf  # [build_platform != target_platform]
+    - oneloop-static  # [build_platform != target_platform]
+    - iregi-static 1.1.0.*  # [build_platform != target_platform]
+    - qcdloop-fortran-static  # [build_platform != target_platform]
+    - ninja-hep-ph-static 1.1.*  # [(build_platform != target_platform) and linux and (x86_64 or ppc64le)]
+    - collier-static  # [build_platform != target_platform]
```

Comment:
```suggestion
```

@chrisburr The `host` requirement versions of these libraries are just providing existing targets for symlinks. If I'm not actively linking against them during the builds of the vendored CutTools and StdHEP static libraries (c.f. [our discussion on the conda-forge Zulip](https://conda-forge.zulipchat.com/#narrow/channel/457337-general/topic/.E2.9C.94.20Acceptable.20to.20add.20a.20fork's.20modifications.20as.20an.20ouput.3F/near/499663264) for context) then I think that I can actually remove all of these and still have the cross-compile work correctly.

Do you agree?

Reply:
Yes you probably don't need them. Generally you only need things to be copied into build if you need them to work during the build, i.e.

* Python and any imports you use during the build (due to conda-forge doing trickery to hack the x86 python install to think it's the aarch64 one)
* You need to use the package to build the package (normally compilers, e.g. `root` needs `rootcling` to work so you have to build `rootcling` for x86 and then use that `rootcling` to build `root` for aarch64)

Reply:
Thanks! Those are useful examples too. :+1: